recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,3 @@
 variant:
   - novtk
-  - all
\ No newline at end of file
+  - all  #[not ppc64le]
```

Comment:
for reference ppc64le was dropped on vtk in https://github.com/conda-forge/vtk-feedstock/pull/335